src/Workspaces/Core/Portable/FindSymbols/FindReferences/Finders/AbstractReferenceFinder.cs

```diff
@@ -590,13 +590,18 @@ protected static bool Matches(ISymbol? symbol1, ISymbol notNullOriginalUnreduced
             var semanticFacts = state.SemanticFacts;
             var semanticModel = state.SemanticModel;
 
-            if (syntaxFacts.IsInNamespaceOrTypeContext(node))
```

Comment:
bug was looking at the node directly, and not walking up it if it was a qualified name.